src/games.py

```diff
@@ -59,35 +85,115 @@ async def _wait_until_next_announcement():
 
         await asyncio.sleep(next_announcement.total_seconds())
 
+    def _add_epic_games(self):
+        games_to_add = self._get_epic_games()
+        existing_games = db.get_games()
+
+        for game_to_add in games_to_add:
+            name, url = game_to_add["name"], game_to_add["url"]
+            if not contains_substring(url, existing_games):
+                # eg_print(f"{name}: added to next announcement")
+                db.add_game(f"{name}: <https://store.epicgames.com/en-US/p/{url}>")
```

Comment:
Some questions:

1. You say in a comment earlier that we aren't going to add auto-fetched Epic games to the db, is this not true?
2. Would it be possible/faster at all to use sets instead to check for duplication?

Reply:
1. Ah sorry, I wasn't very clear. I meant to say that once it has posted epic games for a day, it does not remember it posted those games that day. In other words, they're added to the db for the next announcement, but after that announcement the db is cleared. By not in db I meant "not in another db table that tracks retrieved/posted epic games for that day that is persisted across announcements for the whole day so that if there are multiple announcements in one day they're not double posted". LMK if this still doesn't make sense/I should clarify the comment
2. I don't think so. Sets do equality comparison, and I wanted it to do substring comparison rather than equality comparison of all games for the case where an admin adds a game like "<url>" and the bot goes to post "<name>: <url>". If it was equality comparison we'd get both of those, but it'd be duplicated.